front/migrations/20241205_update_space_group_names.ts

```diff
@@ -0,0 +1,44 @@
+import _ from "lodash";
+
+import { Workspace } from "@app/lib/models/workspace";
+import { SpaceResource } from "@app/lib/resources/space_resource";
+import { makeScript } from "@app/scripts/helpers";
+import { Authenticator } from "@app/lib/auth";
+
+async function backfillWorkspacesGroup(execute: boolean) {
+  const workspaces = await Workspace.findAll();
+
+  const chunks = _.chunk(workspaces, 16);
+  for (const [i, c] of chunks.entries()) {
+    console.log(
+      `[execute=${execute}] Processing chunk of ${c.length} workspaces... (${
+        i + 1
+      }/${chunks.length})`
+    );
+    await Promise.all(
```

Comment:
`nit`
Let's avoid `Promise.all()` => unbounded number of promises and use `concurrentExecutor` instead.

Reply:
here, migration script so that's fine + 16 promises max :)

Reply:
(but rendered moot by runAllWorkspaces anyways :+1: